xml/System.Diagnostics/Activity.xml

```diff
@@ -248,8 +248,8 @@
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>An operation name is the COARSEST name that is useful grouping/filtering. The name is typically a compile-time constant. Names of Rest APIs are reasonable, but arguments (e.g. specific accounts etc), should not be in the name but rather in the tags.</summary>
```

Comment:
Should be `useful for grouping/filtering`. Also, `COARSEST` would be better in italics than in all caps.

Reply:
```suggestion
        <summary>An operation name is the COARSEST name that is useful for grouping/filtering. The name is typically a compile-time constant. Names of Rest APIs are reasonable, but arguments (e.g. specific accounts etc.), should not be in the name but rather in the tags.</summary>
```

Reply:
I couldn't find a way to make a string italic/bold when it's not markdown. @mairaw is it ok if it remains in all caps? What would be your preference?